recipes/slidingwindow/meta.yaml

```
@@ -0,0 +1,51 @@
+{% set name = "slidingwindow" %}
+{% set version = "0.0.14" %}
+{% set sha256 = "4c52adf24e2dd6786b9c1c49144083254d9bd9de655b4f2f17bd78cc41a01e49" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script:
+    - export SETUPTOOLS_SCM_PRETEND_VERSION={{ version }}    # [unix]
+    - "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  host:
+    - python
+    - pip
+
+  run:
+    - python
+    - {{ pin_compatible('numpy') }}
+    - psutil
+
+test:
+  imports:
+    - slidingwindow
+
+about:
+  home: https://github.com/adamrehn/slidingwindow
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE
+  summary: 'Sliding Window library for image processing in Python'
+  description: |
+    This is a simple little Python library for computing a set of windows into a
+    larger dataset, designed for use with image-processing algorithms
+    that utilise a sliding window to break the processing up into a
+    series of smaller chunks. In addition, a set of optional transformations
+    can be specified to be applied to each window.
+
+  doc_url: https://github.com/adamrehn/slidingwindow/blob/master/README.md
+  dev_url: https://github.com/adamrehn/slidingwindow
+
+extra:
+  recipe-maintainers:
+    - adamrehn
```

As @isuruf explained, this line should be at least you and not the developer of this project
```suggestion
    - adamrehn
    - henrykironde
```